This is a bug with the waylock 0.6.5 release tarball. The Git tag for 0.6.5
does not have this problem. This patch can be removed on the next release.

diff -upr waylock-0.6.5.orig/build.zig waylock-0.6.5/build.zig
--- waylock-0.6.5.orig/build.zig	2024-03-03 10:47:47.439903112 +0100
+++ waylock-0.6.5/build.zig	2024-03-03 10:47:55.963263085 +0100
@@ -10,7 +10,7 @@ const Scanner = @import("deps/zig-waylan
 /// development with the "-dev" suffix.
 /// When a release is tagged, the "-dev" suffix should be removed for the commit that gets tagged.
 /// Directly after the tagged commit, the version should be bumped and the "-dev" suffix added.
-const version = "0.6.6-dev";
+const version = "0.6.5";
 
 pub fn build(b: *Build) !void {
     const target = b.standardTargetOptions(.{});
